src/main/java/org/folio/circulation/domain/policy/FixedDueDateSchedules.java

```diff
@@ -47,13 +48,21 @@ static FixedDueDateSchedules from(JsonObject representation) {
 
   private Predicate<? super JsonObject> isWithin(DateTime date) {
     return schedule -> {
-      DateTime from = DateTime.parse(schedule.getString("from"));
-      DateTime to = DateTime.parse(schedule.getString("to"));
+      LocalDate from = DateTime.parse(schedule.getString("from")).toLocalDate();
+      LocalDate to = DateTime.parse(schedule.getString("to")).toLocalDate();
 
-      return date.isAfter(from) && date.isBefore(to);
+      return !isDateOutOfRange(from, to, date.toLocalDate());
```

Comment:
Why has this check been inverted?

Reply:
I used inversion to exclude additional comparison on the equality of dates in the boundary values.

Reply:
Is that because the after should be on or after?

Personally I find the non-inverted logic easier to read